### PR TITLE
migrate(docs): troubleshooting 1

### PR DESCRIPTION
**Pull request by charislam**

### CODE REVIEW

File: apps/docs/content/troubleshooting/why-is-my-camelcase-name-not-working-in-postgres-functions-or-rls-policies-EJMzVd.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title = "Why is my camelCase name not working in Postgres functions or RLS policies?"
+github_url = "https://github.com/orgs/supabase/discussions/12893"
+date_created = "2023-03-08T16:26:51+00:00"
+topics = ["database"]
+---
+
+Avoid camelCase (upper case letters) if it all possible in Postgres for naming of functions, tables, columns, vars. You must enclose "camelCase" in double quotes when you define it or use it. YOU WILL FORGET. Use snake_case instead and make your life easier.
```
Comment (fsansalvadore) [inline apps/docs/content/troubleshooting/why-is-my-camelcase-name-not-working-in-postgres-functions-or-rls-policies-EJMzVd.mdx:8]: at?
```suggestion
Avoid camelCase (upper case letters) at all if possible in Postgres for naming functions, tables, columns, vars. You must enclose "camelCase" in double quotes when you define it or use it. YOU WILL FORGET. Use snake_case instead and make your life easier.
```

Comment (fsansalvadore) [reply apps/docs/content/troubleshooting/why-is-my-camelcase-name-not-working-in-postgres-functions-or-rls-policies-EJMzVd.mdx:8]: Or even removing "at all" (if _at all_ was what you wanted to write)

**Review by fsansalvadore — CHANGES_REQUESTED** [review]

File: apps/docs/content/troubleshooting/why-is-my-camelcase-name-not-working-in-postgres-functions-or-rls-policies-EJMzVd.mdx (hunk 1 above)
Comment (charislam) [reply apps/docs/content/troubleshooting/why-is-my-camelcase-name-not-working-in-postgres-functions-or-rls-policies-EJMzVd.mdx:8]:
```suggestion
Avoid camelCase (upper case letters) if possible when naming functions, tables, columns, and variables. You must enclose "camelCase" in double quotes when you define it or use it. YOU WILL FORGET. Use snake_case instead and make your life easier.
```

**Review by fsansalvadore — APPROVED** [review]
👍👍👍

**Pull request merged**